src/applications/personalization/profile360/selectors.js

```diff
@@ -29,8 +29,15 @@ export const directDepositAddressIsSetUp = state => {
 };
 
 export const directDepositIsBlocked = state => {
-  const controlInfo = directDepositInformation(state)?.responses?.[0]
-    ?.controlInformation;
-  if (!controlInfo) return false;
-  return controlInfo.canUpdateAddress !== true;
+  const controlInfo =
+    directDepositInformation(state)?.responses?.[0]?.controlInformation || {};
+  // If all of these flags are not strictly equal to `true` we want to block
+  // access to the direct deposit feature. ie, we want to err on the side of
+  // blocking people from accessing direct deposit rather than err on the side
+  // of letting people have access to the feature.
+  return (
+    controlInfo.isCompetentIndicator !== true ||
```

Comment:
Can we check here for `false` values instead? 😄

Reply:
Looking at your unit tests, is it either `true` or `null`?

Reply:
Can we instead write this? It's just a bit hard to follow otherwise

`!controlInfo.isCompetentIndicator || !controlInfo.noFiduciaryAssignedIndicator || !controlInfo.notDeceasedIndicator`

Reply:
this code actually came from a previous commit and at the time I, for some reason, wanted to make sure each value was strictly true rather than checking for truthy or not falsey. I don't remember why. But the tests still pass with the simpler check you suggested and I'm happy with this tests. So 👍

Reply:
Awesome! 👍